React/Views/RCTActivityIndicatorViewManager.m

```diff
@@ -23,13 +23,30 @@ @implementation RCTConvert (UIActivityIndicatorView)
 
 @end
 
+@interface RCTActivityIndicatorView : UIActivityIndicatorView
```

Comment:
Let's move this to it's own file for consistency with other views / view managers.